PSICover/PSICover.cs

```diff
@@ -164,30 +167,98 @@ class Analyzer {
          for (int i = 0; i < code.Length; i++)
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
          foreach (var block in blocks) {
+            ulong bID = hits[block.Id];
             bool hit = hits[block.Id] > 0;
-            string tag = $"<span class=\"{(hit ? "hit" : "unhit")}\">";
-            code[block.ELine] = code[block.ELine].Insert (block.ECol, "</span>");
-            code[block.SLine] = code[block.SLine].Insert (block.SCol, tag);
+            string tag = $"<span class=" + (hit ? $"\"hit\" title= \"No.of hits = {bID}\"" : "\"unhit\"") + ">";
+            for (int l = block.SLine; l <= block.ELine; l++) {
+               code[l] = code[l].Insert (code[l].Length, "</span>");
+               code[l] = code[l].Insert (code[l].TakeWhile(char.IsWhiteSpace).Count(), tag);
+            }

```

Comment:
Try the code with single statement like `int i = 0; double d = 0; string str = "";` For first and last line, you should consider SCol and ECol respectively and for other lines you have to consider as in the code.